server/server.js

```diff
@@ -11,5 +12,9 @@ app.use(express.static(path.join(__dirname, '../client/dist')));
 app.listen(1337, () => { console.log('Serving up pictures on port 1337!') });
 
 app.get('/photos', (req, res) => {
-    res.send('GET request worked!')
+    console.log('Express Get Success', res.data);
+    mongodb.Photo.find()
+    // .limit(4)
+    // .sort('title')
+    .then(photos => res.send(photos));
```

Comment:
Response doesn't end

Reply:
Reviewed with you on res.end vs res.send vs res.json! Thanks!